server/sonar-server/src/main/java/org/sonar/server/settings/ws/SetAction.java

```diff
@@ -167,17 +256,25 @@ private void checkType(SetRequest request, PropertyDefinition definition) {
   }
 
   private static void checkValueIsSet(SetRequest request) {
-    checkRequest(isNullOrEmpty(request.getValue()) ^ request.getValues().isEmpty(),
-      "Either '%s' or '%s' must be provided, not both", PARAM_VALUE, PARAM_VALUES);
+    checkRequest(!isNullOrEmpty(request.getValue())
```

Comment:
`One and only one of 'value', 'values', 'fieldValues' must be provided` address the issue